bears/general/URLBear.py

```diff
@@ -18,6 +18,26 @@ class LINK_CONTEXT(Flag):
     pip_vcs_url = 2
 
 
+class URLResult(HiddenResult):
+
+    def __init__(self, line_number, link, code, context):
```

Comment:
see https://github.com/coala/coala/issues/4454 .

The first two parameters should be `Result.origin` and `Result.affected_code` (instead of `line_number`).

I suspect that in the future `URLResult` would expose more of the `Result` base parameters; e.g. it could also use `severity` and `confidence` to expose URLs which probably are not real URLs (possibly replacing `context`).

`code` should be more descriptive `http_status` or `http_status_code`.